packages/toolkit/src/srcShared/fs.ts

```diff
@@ -132,7 +132,15 @@ export class FileSystemCommon {
             return
         }
 
-        return fs.delete(path, { recursive: true })
+        try {
+            await fs.delete(path, { recursive: true })
+        } catch (e) {
+            // The latest vscode version will never throw this error, but the current min version can
+            if (e instanceof vscode.FileSystemError && vscode.FileSystemError.FileNotFound().code === e.code) {
```

Comment:
the `instanceof` check may be counterproductive. if the code looks like a "not found" error, it doesn't really matter what its type is, we want to skip it.
```suggestion
            if (vscode.FileSystemError.FileNotFound().code === e.code) {
```